src/difference.js

```diff
@@ -4,7 +4,7 @@ var _curry2 = require('./internal/_curry2');
 
 /**
  * Finds the set (i.e. no duplicates) of all elements in the first list not
- * contained in the second list.
+ * contained in the second list (in terms of R.contains).
```

Comment:
That's a possibility, is there any circumstance you can imagine where the equality checking is going to be different from that of R.equals? I'd prefer to just say `in terms of R.equals` like `R.contains` does because then users can test out the behaviour themselves and clearly see how it'll behave.